logpush.go

```diff
@@ -0,0 +1,122 @@
+package cloudflare
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// LogpushJob describes a Logpush job
+type LogpushJob struct {
+	ID              int        `json:"id,omitempty"`
+	Enabled         bool       `json:"enabled"`
+	Name            string     `json:"name"`
+	LogpullOptions  string     `json:"logpull_options"`
+	DestinationConf string     `json:"destination_conf"`
+	LastComplete    *time.Time `json:"last_complete,omitempty"`
+	LastError       *time.Time `json:"last_error,omitempty"`
+	ErrorMessage    string     `json:"error_message,omitempty"`
+}
+
+// LogpushJobsResponse is the API response, containing an array of Logpush Jobs.
+type LogpushJobsResponse struct {
+	Success  bool         `json:"success"`
```

Comment:
You should be able to replace `Errors`, `Messages` and `Success` with [`Response`](
https://github.com/cloudflare/cloudflare-go/blob/master/cloudflare.go#L332)